account_invoice_send_mail_queued/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,20 @@
+This module needs no special configuration. However, the module 'queue_job' needs some configuration.
```

Comment:
Shouldn't this point to the `queue_job` README instructions instead?

Reply:
Done